internal/databases/postgres/backup_list_test.go

```diff
@@ -0,0 +1,223 @@
+package postgres_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/databases/postgres"
+	"github.com/wal-g/wal-g/testtools"
+)
+
+func TestBackupListFlagsFindsBackups(t *testing.T) {
+	folder := testtools.CreateMockStorageFolder()
+	postgres.HandleDetailedBackupList(folder, true, false)
+}
+
+func TestBackupListCorrectOutput(t *testing.T) {
+	const expected = "" +
+	"name   modified             wal_segment_backup_start\n" +
+	"base_2 2020-01-01T01:01:01Z ZZZZZZZZZZZZZZZZZZZZZZZZ\n" +
+	"base_0 2018-01-01T01:01:01Z ZZZZZZZZZZZZZZZZZZZZZZZZ\n" +
+	"base_1 2017-01-01T01:01:01Z ZZZZZZZZZZZZZZZZZZZZZZZZ\n"
+
+	folder := testtools.CreatePostgresMockStorageFolderWithTimeMetadata(t, testtools.NoCreationTime)
+	backups, err := internal.GetBackups(folder)
+	assert.NoError(t, err)
+	buf := new(bytes.Buffer)
+	internal.WriteBackupList(backups, buf)
+	assert.Equal(t, buf.String(), expected)
+}
+
+func TestBackupListCorrectPrettyOutput(t *testing.T) {
+	const expected =
+	"+---+--------+-----------------------------------+--------------------------+------------+-------------+----------+---------+------------+-----------+------------+-----------+\n" +
+	"| # | NAME   | MODIFIED                          | WAL SEGMENT BACKUP START | START TIME | FINISH TIME | HOSTNAME | DATADIR | PG VERSION | START LSN | FINISH LSN | PERMANENT |\n" +
+	"+---+--------+-----------------------------------+--------------------------+------------+-------------+----------+---------+------------+-----------+------------+-----------+\n" +
+	"| 0 | base_2 | Wednesday, 01-Jan-20 01:01:01 UTC | ZZZZZZZZZZZZZZZZZZZZZZZZ | -          | -           |          |         |          0 |         0 |          0 | false     |\n" +
```

Comment:
Since this test is Postgres-specific now, why does `WAL SEGMENT BACKUP START` is `ZZZZZZZZZZZZZZZZZZZZZZZZ`? And other values are absent too.

Maybe fill other fields of metadata.json and tests that they are being displayed too?

If this requires some noticeable effort, we can postpone it to the separate PR and merge the current as it is.